client/card-readers/list/index.tsx

```diff
@@ -9,8 +9,8 @@ import { Card, CardBody, CardDivider } from '@wordpress/components';
 /**
  * Internal dependencies
  */
-import SettingsSection from 'wcpay/settings//settings-section';
-import SettingsLayout from 'wcpay/settings//settings-layout';
+import SettingsSection from 'wcpay/settings/settings-section';
+import SettingsLayout from 'wcpay/settings/settings-layout';
 import LoadableSettingsSection from 'wcpay/settings//loadable-settings-section';
```

Comment:
Can wee replace `//` to `/` here as well?

Reply:
Done